Docs auto update

joaopapereira opened the pull request.

joaopapereira committed:
Allow usage of sha1sum for OS's that do not have sha256sum

Signed-off-by: João Pereira <joao.pereira@broadcom.com>

joaopapereira committed:
Automated documentation release for new minor

Signed-off-by: João Pereira <joao.pereira@broadcom.com>

praveenrewar commented on .github/workflows/published_release.yml:
```diff
@@ -172,7 +172,50 @@ jobs:
+  prepare-documentation-update:
+    runs-on: ubuntu-latest
+    needs: [process-release-information,update-releases-yaml]
+    if: endsWith(github.event.client_payload.tagName, '.0')
```
Will this also include prereleases? Do we want to include them?

joaopapereira replied:
what do you mean by prereleases?

joaopapereira replied:
I do not think that we do prereleases in our tools, but if we do we can look into it later

joaopapereira committed:
Fix issues found

- Ensure the shasum logic is correct
- Update all the version of the actions being used to the latest

Signed-off-by: Joao Pereira <joaopapereira@gmail.com>

joaopapereira committed:
Cache trivy db inside our org

Signed-off-by: Joao Pereira <joaopapereira@gmail.com>

joaopapereira committed:
Fix issue related to ssh keys

Signed-off-by: Joao Pereira <joaopapereira@gmail.com>

joaopapereira committed:
Fix issues:

- shell injection protection
- missing popd
- Variable not set

Signed-off-by: Joao Pereira <joaopapereira@gmail.com>

joaopapereira commented on .github/workflows/published_release.yml:
```diff
@@ -172,22 +172,69 @@ jobs:
+          echo "${{ secrets.INSTALL_SH_DEPLOY_PRIVATE_KEY }}" > "${TMP_DEPLOY_PRIV_KEY}"
+          chmod 600 "${TMP_DEPLOY_PRIV_KEY}"
+          eval $(ssh-agent -s)
+          ssh-add "${TMP_DEPLOY_PRIV_KEY}"
```
fixed
